### DISCUSSION TITLE
Updating in soil parameters for post-failure analysis of landslide

### DISCUSSION THREAD

**Discussion by MiguelSC98** [Anura3D/Anura3D_OpenSource#92] [Q&A] [answered]
Dear all.
Hi.
I want to know if it's possible in Anura3D 2022 (pre processor GID 14.0.1) to update geotechnical parameters (i.g. reduction of friction angle and cohesion) or soil behavior (e.g linear elastic to mohr-couloumb) and what steps I must follow.
I ask this question to define stage 1: Gravity loading with soil parameters in pre-failure conditions and stage 2: Slope failure, by reducing the soil parameters or changing soil behavior.
Thank you in advance for your help.

**Comment by FrancescaCeccato** [comment, marked as answer]
Sure, it is possible.
Run step 1 with original parameters and write results.
Modify the GOM file with the new parameters of the material, activate the flag $$APPLY_MATERIAL_UDATE (MUST BE 1)
then run step 2.

**Comment by zhangrf21** [comment]
Anura3D 2023可以吗
